### ISSUE TITLE
Changing the order of the edges in a network?

### ISSUE THREAD

**Issue by dataquestions** [datastorm-open/visNetwork#271]
Hi, 

How can one change the order of the network edges, 

for example, i would like all the (to) edges together, then all the (from) edges together, then all the (to;from) edges together, when the network is made rather than having the network randomly place the edges in any order, 

so essentially the edges flow around in a circle, in order of to/from/to;from 

haven't found anything online, and have tried arranging the underlying edge data in alphabetical order, but still have not got anywhere, 

any ideas?

**Comment by dataquestions** [comment]
@bthieurmel any ideas>?

**Comment by Dmirijs-Ozernovs** [comment]
You can use _visIgraphLayout_ for layouts from igraph package [(link)](https://www.rdocumentation.org/packages/visNetwork/versions/2.0.4/topics/visIgraphLayout)

With _layout_in_circle_ you can specify order of the vertices [(link)](http://igraph.org/r/doc/layout_in_circle.html)

**Comment by dataquestions** [comment]
can i specify the order of the vertices within the layout_as_star from the igraph network? I would like this as i then could for example have 1 node. which the user has selected in the middle of the layout (essentially in the centre of the network) and then have the surrounding associated nodes ordered by to, from and to;from order

I hope doing it in a layout as a star is possible

**Comment by dataquestions** [comment]
@Dmirijs-Ozernovs  thank you for your response, very much appreciated, so if i was to use the 

http://igraph.org/r/doc/layout_as_star.html

layout as a star, i could then create some sort of additional column which would be essentially be based on the order to which i would like the plot to be made with?

i think i know what you mean, use the order argument within this igraph layout, correct me if i am wrong - but thanks for responding!

**Comment by Dmirijs-Ozernovs** [comment]
> so if i was to use the layout as a star, i could then create some sort of additional column which would be essentially be based on the order to which i would like the plot to be made with?
> i think i know what you mean, use the order argument within this igraph layout, correct me if i am wrong
Yes, _layout.star_ also supports order argument, so you can specify the order you need. Pay attention, this is the order of the vertices, not the edges.
